### ISSUE TITLE
MFPaint: Themes

### ISSUE THREAD

**Issue by Brycey92** [AEFeinstein/Super-2023-Swadge-FW#301]
Changeable UI themes for MFPaint, much like you can change your color schemes for notepad++, IDEs, etc.

**Comment by AEFeinstein** [comment]
Now you're just doing super stretch goals, lol

**Comment by Brycey92** [comment]
Yup! @dylwhich actually said this sounds "easy" to do before event, so ¯\\\_(ツ)_/¯ the madman can have at it
